ttnctl/cmd/v3types.go

```diff
@@ -0,0 +1,104 @@
+// Copyright © 2020 The Things Network
+// Use of this source code is governed by the MIT license that can be found in the LICENSE file.
+
+package cmd
+
+import "fmt"
+
+type v3ApplicationIDs struct {
```

Comment:
If we had https://github.com/TheThingsNetwork/lorawan-stack/pull/2453 we could just import the `ttnpb` types (cc @johanstokking )

Reply:
I don't think we want to import the V3 repo in V2